py2many/inference.py

```diff
@@ -435,26 +445,32 @@ def visit_BinOp(self, node):
                     return node
             node.annotation = left
             return node
-        else:
```

Comment:
This code was covering `i8-i64 binop float` case. Perhaps didn't have test coverage.

Reply:
I believe no code path was changed here.  I only removed the unnecessary dangling else.